collection-scripts/gather_core_dumps

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+CORE_DUMP_PATH=${OUT:-"${BASE_COLLECTION_PATH}/node_core_dumps"}
+
+mkdir -p ${CORE_DUMP_PATH}/
+
+function gather_core_dump_data {
+  for NODE in ${NODES}; do
+    oc debug node/${NODE} -- chroot /host /bin/bash -c "coredumpctl info" > ${CORE_DUMP_PATH}/${NODE}_core_dump & PIDS+=($!)
+  done
+}
+
+PIDS=()
+NODES="${@:-$(oc get nodes --no-headers -o custom-columns=':metadata.name')}"
```

Comment:
We need to scope this to linux only hosts.

Reply:
Agreed, updated accordingly